build.gradle

```diff
@@ -21,16 +23,29 @@ buildscript {
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
         classpath 'gradle.plugin.org.spongepowered:spongegradle:0.8.1'
         classpath 'org.spongepowered:mixingradle:0.4-SNAPSHOT'
+        classpath mcTesterDep
```

Comment:
Hmm, Would you mind pointing out where `mcTesterDep` is defined? I failed to see that.

Reply:
`gradle.properties`

Reply:
Could the dependency be normalized so it's not classpath based? As we will eventually be moving to Kotlin-dsl, classpath provided plugins are annooying since they have to be applied in the old deprecated methods, versus applying via the `plugins {}` format.

More can be read here: https://docs.gradle.org/current/userguide/kotlin_dsl.html#sec:plugins_resolution_strategy